test/run

```diff
@@ -16,7 +16,18 @@ test -n "${IMAGE_NAME-}" || { echo 'make sure $IMAGE_NAME is defined' && false ;
 test -n "${VERSION-}" || { echo 'make sure $VERSION is defined' && false; }
 test -n "${OS-}" || { echo 'make sure $OS is defined' && false; }
 
-test_exit=1
+test_short_summary=''
+TESTSUITE_RESULT=0
+
+TEST_LIST="\
+run_container_creation_tests
+run_tests_no_root
+run_tests_no_pass
+run_tests_no_pass_altuid
+run_tests_no_root_altuid
+run_change_password_test
+run_doc_test
```

Comment:
run_doc_test could return 1, but nothing checks it. I would add check_result into run_all_tests after $test_case